scripts/python/policia_nacional/pdf_to_csv.py

```diff
@@ -0,0 +1,103 @@
+import tabula

```

Comment:
This code is very hard to read, python code is normally formated using the pep8 convention, you can check here tools to format your code: https://stackoverflow.com/questions/14328406/tool-to-convert-python-code-to-be-pep8-compliant